### ISSUE TITLE
Event listener callback not triggering when 'eventInfo' is set on an event

### ISSUE THREAD

**Issue by charltoncraig** [adobe/adobe-client-data-layer#93]
### Expected Behaviour
When an event of the format {'event':_eventName_,'eventInfo':_info_} is pushed into window.adobeDataLayer, any event listener registered to handle this event being pushed should fire it's callback function
### Actual Behaviour
When the 'eventInfo' key is included in the event object, the callback function does not fire
#### Steps to Reproduce
Navigate to about:blank in a browser
Execute the following in the console
```
window.adobeDataLayer = window.adobeDataLayer || []
scriptElement = document.createElement('script');
scriptElement.src="https://unpkg.com/@adobe/adobe-client-data-layer@1.0.0/dist/adobe-client-data-layer.min.js";
document.body.append(scriptElement)

window.adobeDataLayer.push({'event':'newEvent','eventInfo':'test'})

var myHandler = function(event) {
    console.log(event)
}

window.adobeDataLayer.push(function(dl) {
    dl.addEventListener(
        "adobeDataLayer:event", 
        myHandler
    )
})
```
(Optional) Execute the following in the console to see the callback function executing correctly
```
window.adobeDataLayer = window.adobeDataLayer || []
scriptElement = document.createElement('script');
scriptElement.src="https://unpkg.com/@adobe/adobe-client-data-layer@1.0.0/dist/adobe-client-data-layer.min.js";
document.body.append(scriptElement)

window.adobeDataLayer.push({'event':'newEvent','notEventInfo':'test'})

var myHandler = function(event) {
    console.log(event)
}

window.adobeDataLayer.push(function(dl) {
    dl.addEventListener(
        "adobeDataLayer:event", 
        myHandler
    )
})
```
#### Platform and Version
Chrome v83.0.4103.116

**Comment by charltoncraig** [comment]
Potentially fixed in https://github.com/adobe/adobe-client-data-layer/pull/92

**Comment by bartoszglow** [comment]
Hello! Thank you for your input.

This issue has been fixed with version `1.0.2` of Adobe Client Data Layer. When you modify your code snippet and change version of ACDL to `1.0.2` you will get callbacks triggered properly.

Please try to always use the newest version of ACDL to make sure you get the best experience using it.
